mail/tasks.py

```diff
@@ -0,0 +1,75 @@
+"""
+Tasks for mail app
+"""
+import logging
+from decimal import Decimal
+
+from django.conf import settings
+from rest_framework import status
+
+from bootcamp.celery import async
+from ecommerce.models import Line
+from klasses.models import Klass, BootcampAdmissionCache
+from mail.api import MailgunClient
+from mail.models import AutomaticReminderEmail, SentAutomaticEmails
+from mail.utils import generate_mailgun_response_json
+
+log = logging.getLogger(__name__)
+
+
+def send_reminder_payment_emails():
```

Comment:
Why not use `MailgunClient.send_batch` with Mailgun variables instead? I'm concerned about the number of API calls this would make